packages/ember-htmlbars/tests/integration/component_invocation_test.js

```diff
@@ -53,516 +34,6 @@ QUnit.module('component - invocation', {
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{non-block someProp=view.someProp}}'),
-    someProp: 'wycats'
-  }).create();
-
-  runAppend(view);
-
-  equal(didReceiveAttrs, 1, 'The didReceiveAttrs hook fired');
-
-  equal(jQuery('#qunit-fixture').text(), 'In layout - someProp: wycats');
-
-  run(function() {
-    view.set('someProp', 'tomdale');
-  });
-
-  equal(jQuery('#qunit-fixture').text(), 'In layout - someProp: tomdale');
-  equal(didReceiveAttrs, 2, 'The didReceiveAttrs hook fired again');
-  equal(willUpdate, 1, 'The willUpdate hook fired once');
-
-  run(view, 'rerender');
-
-  equal(jQuery('#qunit-fixture').text(), 'In layout - someProp: tomdale');
-  equal(didReceiveAttrs, 3, 'The didReceiveAttrs hook fired again');
-  equal(willUpdate, 2, 'The willUpdate hook fired again');
-});
-
-
-QUnit.test('[DEPRECATED] non-block with properties on self', function() {
-  // TODO: attrs
-  // expectDeprecation("You accessed the `someProp` attribute directly. Please use `attrs.someProp` instead.");
-
-  owner.register('template:components/non-block', compile('In layout - someProp: {{someProp}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{non-block someProp="something here"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'In layout - someProp: something here');
-});
-
-QUnit.test('block with properties on attrs', function() {
-  expect(1);
-
-  owner.register('template:components/with-block', compile('In layout - someProp: {{attrs.someProp}} - {{yield}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{#with-block someProp="something here"}}In template{{/with-block}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'In layout - someProp: something here - In template');
-});
-
-QUnit.test('[DEPRECATED] block with properties on self', function() {
-  // TODO: attrs
-  // expectDeprecation("You accessed the `someProp` attribute directly. Please use `attrs.someProp` instead.");
-
-  owner.register('template:components/with-block', compile('In layout - someProp: {{someProp}} - {{yield}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{#with-block someProp="something here"}}In template{{/with-block}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'In layout - someProp: something here - In template');
-});
-
-QUnit.test('with ariaRole specified', function() {
-  expect(1);
-
-  owner.register('template:components/aria-test', compile('Here!'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{aria-test id="aria-test" ariaRole="main"}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$('#aria-test').attr('role'), 'main', 'role attribute is applied');
-});
-
-QUnit.test('`template` specified in a component is overridden by block', function() {
```

Comment:
never mind, I think this is :heavy_check_mark: `template` specified in component is overriden by block